ons-spark/raw-notebooks/checkpoint-staging/r_input.R

```diff
@@ -0,0 +1,150 @@
+options(warn = -1) 
+library(sparklyr)
+library(dplyr)
+library(DBI)
+
+sc <- sparklyr::spark_connect(
+    master = "local[2]",
+    app_name = "checkpoint",
+    config = sparklyr::spark_config())
+
+
+set.seed(42)
+new_cols <- 12
+num_rows <- 10^3
+
+start_time <- Sys.time()
+
+df = sparklyr::sdf_seq(sc, 1, num_rows) %>%
+    sparklyr::mutate(col_0 = ceiling(rand()*new_cols))
+
+for (i in 1: new_cols)
+{
+  column_name = paste0('col_', i)
+  prev_column = paste0('col_', i-1)
+  df <- df %>%
+    mutate(
```

Comment:
need to reference the package for each R function (unless it's base R). I think the way Adrian suggested we do this is to develop the R code by just importing the `magrittr` package and nothing else, this will force you to reference each package, e.g. `dplyr::groupby()`. Then at the end you can get rid of the `magrittr` import and put `sparklyr` and `dplyr` back in. Take a look at other .r files in the repo as examples.